modules/javafx.base/src/main/java/javafx/beans/property/Property.java

```diff
@@ -40,76 +40,95 @@
+     * to the value of one property will immediately result in the value of the other property being
+     * changed accordingly.
      * <p>
-     * JavaFX bidirectional binding implementation use weak listeners. This means bidirectional binding does not prevent
-     * properties from being garbage collected.
+     * While it is not possible for a property to be bound by more than one unidirectional binding,
+     * it is legal to establish multiple bidirectional bindings for the same property. However,
+     * since a bidirectional binding allows for the values of both properties to be changed
+     * by calling {@link #setValue(Object)}, neither of the properties is considered to be a
+     * <i>bound property</i> that returns {@code true} from {@link #isBound()}.
+     * <p>
+     * Both properties of a bidirectional binding <i>weakly</i> reference their counterparts.
+     * This is different from a unidirectional binding, where the bound property <i>strongly</i>
+     * references its binding source. In practice, this means that if any of the bidirectionally
+     * bound properties become unreachable, the binding is eligible for garbage collection.
+     * Furthermore, neither of the bidirectionally bound properties will keep its counterpart
+     * alive if the counterpart would otherwise become unreachable.
+     * <p>
+     * Bidirectional bindings are independent from unidirectional bindings. As a consequence,
+     * establishing a bidirectional binding does not remove a unidirectional binding that has
+     * already been established, nor does it prevent a unidirectional binding from being
+     * established. However, doing so is not a meaningful use of this API, because the
+     * unidirectionally bound property will cause any attempt to change the value of any of
+     * the properties to fail with an exception.
      *
-     * @param other
-     *            the other {@code Property}
-     * @throws NullPointerException
-     *             if {@code other} is {@code null}
-     * @throws IllegalArgumentException
-     *             if {@code other} is {@code this}
+     * @param other the other property
+     * @throws NullPointerException if {@code other} is {@code null}
+     * @throws IllegalArgumentException if {@code other} is {@code this}
      */
     void bindBidirectional(Property<T> other);
 
     /**
-     * Remove a bidirectional binding between this {@code Property} and another
-     * one.
-     *
-     * If no bidirectional binding between the properties exists, calling this
-     * method has no effect.
-     *
-     * It is possible to unbind by a call on the second property. This code will work:
-     *
-     * <blockquote><pre>
-     *     property1.bindBirectional(property2);
-     *     property2.unbindBidirectional(property1);
-     * </pre></blockquote>
+     * Removes a bidirectional binding that was established with {@link #bindBidirectional(Property)}.
```

Comment:
"the" (it's possible to have multiple bidi bindings, but only 1 to the same `other` property).